src/metrics/enhanced-metrics.ts

```diff
@@ -48,7 +49,14 @@ export function getEnhancedMetricTags(context: Context): string[] {
   if (context.invokedFunctionArn) {
     arnTags = parseTagsFromARN(context.invokedFunctionArn, context.functionVersion);
   }
-  const tags = [...arnTags, getColdStartTag(), `memorysize:${context.memoryLimitInMB}`];
+  const ddtraceVersion = packageJson.devDependencies["dd-trace"];
+  const tags = [
+    ...arnTags,
+    getColdStartTag(),
+    `memorysize:${context.memoryLimitInMB}`,
+    `datadog_lamda:${packageJson.version}`,
+    `dd_trace:${ddtraceVersion}`,
```

Comment:
These are for enhanced metrics and not for traces. I think we need the library version in the enhanced metric, but not sure if `dd_trace` is required here as it's not really used (AFAIK)

Reply:
added tags for dd_trace as well